erizo/src/test/SdpTest.cpp

```diff
@@ -1,253 +0,0 @@
-#include <gmock/gmock.h>
```

Comment:
👍 I'm pretty sure we don't need the sdps that we're storing in `src/test/assets` since we're not going to test the parsing right?

Reply:
good catch!